README.md

```diff
@@ -7,7 +7,7 @@
 ![Awesome badge](images/awesome.svg)
 
 :wave: Welcome to **Python3 by practice**! We are an open source community contributing to compile a collection of learning resources
-for CSE/IT enthusiasts, or anyone who finds these resources useful to learn Python 3. If you contributed to this repo in any manner 
+for **Python** and **Machine Learning** enthusiasts, or anyone who finds these resources useful. If you contributed to this repo in any manner 
```

Comment:
It would be better to have just "Python" as this repo aims to help over a variety of topics and not just "Machine Learning with Python". With this, I also think bolding them would not be required.

Reply:
ok, I'll do it.